static/js/theme-map/VerticalFullPageMapOrchestrator.js

```diff
@@ -157,21 +159,28 @@ class VerticalFullPageMapOrchestrator extends ANSWERS.Component {
     const pinClusterClickListener = () => this.searchOnMapMove && this.searchThisArea();
 
     /**
-     * Dragging the map searches the new area, if desired
-     *
-     * @param {Map} map The map object
+     * The listener called when the map drag ends
      */
-    const dragEndListener = () => this.handleMapAreaChange();
+    const dragEndListener = () => {};
```

Comment:
Do we sill need the `dragEndListener`?

Reply:
nope. I left it in here so that it would be easy to modify if we ever wanted to change it, however I could remove it if you'd prefer

Reply:
If it's not used right now, I think we should remove. We can add it back in if we ever need it.